### DISCUSSION TITLE
Problems separating _site to deploy branch

### DISCUSSION THREAD

**Discussion by andrewmontoyaiv** [community/community#23316] [Other Feature Feedback, Questions, & Ideas]
<p>I’m fairly new to using git/jekyll, so pardon my ignorance. I am using some plugins for my jekyll site, and it’s causing some things to not render properly. Specifically I am using paginate-v2, and I suspect the inclusion of paginator.posts in my blog page is causing posts to not render (since it’s tied to the plugin and renders fine locally). I followed <a href="http://sarahcassady.com/2015/07/17/jekyll-on-github-pages/" rel="nofollow noopener">this article</a> for separating _site and source code into master/source branches, but with just having _site in the master branch, github doesnt recognize the repo as a website repo. Am I doing something wrong? Is it possible to get github pages to only consider _site when building remotely?</p>

**Comment by andrewmontoyaiv** [comment, marked as answer]
<p>Nevermind, got it working by moving _site content into the root directory &amp; adding .nojekyll</p>

**Comment by smaroukis** [comment]
<p>I’m trying to get this working too and even with all the _site contents in root with a .nojekyll file I don’t think it is serving properly. What do you see under Environments / Deployment in your repo? Mine shows an Abandoned deployment in the activity log</p>

**Comment by smaroukis** [comment]
<p>Working now all good</p>
